### PR TITLE
Fix CI jobs for ironic containers

### PR DESCRIPTION
**Pull request by elfosardo**
Includes base image fixes for OCP 4.13, 4.14 and 4.15

### CODE REVIEW

**Comment by elfosardo** [comment]
/pj-rehearse ack
good enough

File: ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-release-4.15.yaml
Comment (iurygregory) [inline ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-release-4.15.yaml:1]: the file here mentions 4.15... for me it's ok to use 4.14, but I want to double check

**Review by iurygregory — APPROVED** [review]

File: ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-release-4.15.yaml
Comment (elfosardo) [reply ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-release-4.15.yaml:1]: this is intended, as we're currently in the development cycle for 4.14, we have the 4.15 configuration also ready, using the available 4.14 images

**Pull request merged**